File: js/programmers/level.0/prgm_120876_1.js
function solution(lines) {
  let result = 0;
  let lineMap = new Array(200).fill(0);

  for (i = 0; i < 3; i++) {
    let left = lines[i][0];
    let right = lines[i][1];

    for (j = left; j < right; j++) {
      lineMap[j + 100]++;
    }
  }

  for (i in lineMap) {
    if (lineMap[i] > 1) {
      result++;
    }
  }

  return result;
}

File: python/programmers/level.0/prgm_120876_1.py
def solution(lines):
    result = 0
    line_map = [0 for _ in range(200)]

    for i in range(3):
        left = lines[i][0]
        right = lines[i][1]

        for j in range(left, right):
            line_map[j+100] += 1

    for i in range(len(line_map)):
        if line_map[i] > 1: result += 1

    return result